releasenotes/notes/0.46/prepare-0.46.0-c3857463a3c052f7.yaml

```
@@ -0,0 +1,625 @@
+    legacy parser is removed. If you were previously using this, you can
+    instead parse your OpenQASM 2 files into a :class:`.QuantumCircuit` using
+    the :meth:`.QuantumCircuit.from_qasm_file` or
+    :meth:`.QuantumCircuit.from_qasm_str` constructor methods and then
+    converting that :class:`.QuantumCircuit` into a :class:`.DAGCircuit`
+    with :func:`.circuit_to_dag`.
+  - |
+    The :meth:`.QuantumCircuit.qasm` method used to generate a OpenQASM 2
+    representation of the :class:`.QuantumCircuit` object has been deprecated
+    and will be removed in the Qiskit 1.0.0 release. The
+    :func:`.qasm2.dump` or :func:`.qasm2.dumps` functions which provide similar
+    functionality should be used instead. If you were using the :meth:`.QuantumCircuit.qasm`
+    method to generate pygments formatted output you should instead look at the standalone
+    ``openqasm-pygments`` package to provide this functionality (as :func:`.qasm2.dump`
+    and :func:`.qasm2.dumps` do not provide pygments colored output).
+
+  - |
+    The :class:`~qiskit.pulse.library.parametric_pulses.ParametricPulse` base class and pulses are now
+    deprecated, and will be removed in Qiskit 1.0. This includes:
+
+    * :class:`~qiskit.pulse.library.parametric_pulses.ParametricPulse`
+    * :class:`~qiskit.pulse.library.parametric_pulses.Constant`
+    * :class:`~qiskit.pulse.library.parametric_pulses.Drag`
+    * :class:`~qiskit.pulse.library.parametric_pulses.Gaussian`
+    * :class:`~qiskit.pulse.library.parametric_pulses.GaussianSquare`
+
+    The class has been superseded by :class:`~qiskit.pulse.SymbolicPulse` and the corresponding pulse library.
+    :class:`~qiskit.pulse.SymbolicPulse` provides better performance, flexibility and QPY support.
+  - |
+    The :class:`.NoiseAdaptiveLayout` transpiler pass and the corresponding
+    ``"noise_adaptive"`` layout stage plugin have been deprecated and will
+    be removed in the 1.0.0 release. This pass has been largely superseded by
+    :class:`.VF2Layout` and :class:`.VF2PostLayout` which will set a layout
+    based on the reported noise characteristics of a backend.
+  - |
+    The :class:`.CrosstalkAdaptiveSchedule` transpiler pass has been deprecated
+    and will be removed in the 1.0.0 release. This pass was not usable any
+    longer because its internal operation was dependent on custom properties
+    being set in the :class:`.BackendProperties` payload of a
+    :class:`.BackendV1` instance. As no backends are setting these fields
+    the pass has been deprecated.
+  - |
+    The :mod:`qiskit.visualization.qcstyle` module is now deprecated and will be removed
+    in the Qiskit 1.0.0 release. Instead you should use
+    :mod:`qiskit.visualization.circuit.qcstyle` as direct replacement.
+
+  - |
+    Injecting circuit gate operations into the pulse builder context is now deprecated. The deprecation
+        affects the following functions:
```

Not sure if this indentation has a Sphinx meaning, if not, feel free to ignore
```suggestion
    affects the following functions:
```

---

I think that might have been from my previous inline suggestion (so it can go).